pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[tool.poetry]
+name = "planemo"
+version = "0.75.0"
+description = "Command-line utilities to assist in building tools for the Galaxy project (http://galaxyproject.org/)."
+authors = ["Galaxy Project and Community"]
+license = "MIT"
+
+[tool.poetry.dependencies]
+python = "^3.6"
+allure-python-commons = "*"
+BeautifulSoup4 = "*"
+bioblend = "*"
+click = "*"
+configparser = "*"
+cwltool = "*"
+docutils = "*"
+ephemeris = "*"
+galaxy-containers = "*"
+galaxy-tool-util = "*"
+galaxy-util = "*"
+glob2 = "*"
+gxformat2 = "*"
+jinja2 = "*"
+lxml = "*"
+oyaml = "*"
+pyaml = "*"
+pyyaml = "*"
+six = "*"
+virtualenv = "*"
+tabulate = "*"
+
+[tool.poetry.dev-dependencies]
+toil = "*"
+flask = "*"
+Werkzeug = "*"
+tox = "*"
+pytest = "*"
+coverage = "*"
+sphinx = "*"
+sphinx_rtd_theme = "*"
+recommonmark = "*"
+readme = "*"
+pyflakes = "*"
+flake8 = "*"
+flake8-import-order = "*"
+wheel = "*"
+twine = "*"
+
+[build-system]
+requires = ["poetry-core>=1.0.0", "setuptools"]
+build-backend = "poetry.core.masonry.api"
```

Comment:
I don't think we want to force the use of `poetry` to install planemo.

```suggestion
```

Reply:
I do think we need `requires = ["setuptools"]`, otherwise it seems `pip install -e` fails

Reply:
I see, that's only needed if we have the `[build-system]` section